python/CMakeLists.txt

```diff
@@ -411,6 +411,8 @@ if (PYARROW_BUILD_PARQUET)
   endif()
   include_directories(SYSTEM ${PARQUET_INCLUDE_DIR})
 
+  file(COPY ${PARQUET_INCLUDE_DIR}/parquet DESTINATION ${BUILD_OUTPUT_ROOT_DIRECTORY}/include)
```

Comment:
Can you guard this with a check for `if (PYARROW_BUNDLE_ARROW_CPP)…`?

We should only include those headers when we are also the source of `libparquet.so`.